src/config.ts

```diff
@@ -0,0 +1,107 @@
+
+
+export class Configuration {
+
+    /**
+    * Browser-specific constant to adjust CanvasRenderingContext2D.shadowBlur value,
+    * which is unitless and not rendered equally across browsers.
+    *
+    * Values that work quite well (as of October 2017) are:
+    * - Chrome: 1.5
+    * - Edge: 1.75
+    * - Firefox: 0.9
+    * - Safari: 0.95
+    *
+    * @since 2.0.0
+    * @type Number
+    * @default 1
+    */
+    browserShadowBlurConstant = 1
+
+
+    /**
+     * Pixel per Inch as a default value set to 96. Can be changed for more realistic conversion.
+     */
+    DPI = 96
+
+    /**
+     * Pixel limit for cache canvases. 1Mpx , 4Mpx should be fine.
+     * @since 1.7.14
+     * @type Number
+     * @default
+     */
+    perfLimitSizeTotal = 2097152
+
+    /**
+     * Pixel limit for cache canvases width or height. IE fixes the maximum at 5000
+     * @since 1.7.14
+     * @type Number
+     * @default
+     */
+    maxCacheSideLimit = 4096
+
+    /**
+     * Lowest pixel limit for cache canvases, set at 256PX
+     * @since 1.7.14
+     * @type Number
+     * @default
+     */
+    minCacheSideLimit = 256
+
+    /**
+     * if webgl is enabled and available, textureSize will determine the size
+     * of the canvas backend
+     * @since 2.0.0
+     * @type Number
+     * @default
+     */
+    textureSize = 2048
```

Comment:
we should bump this to 4096.
there was ie11 on some kind of intel gpu that was freaking out at more than 2248.
We can leave a note that if they want to support older hardware devs should pull this down to 2048

Reply:
DONE